.github/workflows/quickstart_microprofile-rest-client_ci.yml

```diff
@@ -0,0 +1,18 @@
+name: WildFly microprofile-rest-client Quickstart CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches-ignore:
+      - 'dependabot/**'
+    paths:
+      - 'microprofile-rest-client/**'
+      - '.github/workflows/quickstart_ci.yml'
+
+jobs:
+  call-quickstart_ci:
+    uses: ./.github/workflows/quickstart_ci.yml
+    with:
+      QUICKSTART_PATH: microprofile-rest-client
+      TEST_PROVISIONED_SERVER: true
```

Comment:
please remove this line, mp quickstarts should only use bootable-jar only as server provisioning (to be changed in the future when wildfly jar plugin supports bootable jar packaging)